README.md

```diff
@@ -22,7 +22,7 @@ To use CometDClient in your project add the following 'Podfile' to your project
   platform :ios, '10.0'
   use_frameworks!
 
-pod 'CometDClient', '~> 1.0.0'
+    pod 'CometDClient', '~> 1.0.0'
```

Comment:
Sorry it was not very clear 😄 thanks to github to be able to select multiple lines. So maybe it's this entier code that you need to format to a code block because it the thing that the user will copy to is project.

Reply:
<img width="698" alt="Capture d’écran 2020-05-20 à 12 00 56" src="https://user-images.githubusercontent.com/16764808/82433319-90e3b900-9a91-11ea-9e7a-f350e9b733ab.png">
For me it's ok ?

Reply:
```source 'https://github.com/CocoaPods/Specs.git'
  platform :ios, '10.0'
  use_frameworks!
```